main.py

```
@@ -1,20 +1,310 @@
+        await q.message.reply_text(text, reply_markup=back_button(True))
+
+    elif q.data == "user_stats" and is_admin(uid):
+        users = load_users()
+        await q.message.reply_text(f"👥 Total Users: {len(users)}", reply_markup=back_button(True))
+
+    elif q.data == "close_this_chat" and uid == OWNER_ID:
+        try:
+            target_uid = int(q.message.text.split("user ")[1].split(":")[0].strip())
+            closed = load(CLOSED_CHATS_DB)
+            if target_uid not in closed:
+                closed.append(target_uid); save(CLOSED_CHATS_DB, closed)
```

Ruff (0.14.10)

Multiple statements on one line (semicolon)

(E702)